src/Graph5/Graph5.tsx

```diff
@@ -0,0 +1,46 @@
+import { AreaChart, Area, ResponsiveContainer } from "recharts";
+import data from "./DataGraph5.json";
+
+function findHighestUV(data: { name: string; uv: number }[]): number {
+	const uvValues = data.map((item) => item.uv);
+	const highestUV = Math.max(...uvValues);
+	return highestUV;
+}
+
+const highestUVValue = findHighestUV(data);
+
+export function Graph5() {
+	return (
+		<div>
+			<h1 className="text-2xl  font-bold">{highestUVValue}</h1>
```

Comment:
Te h1 zmien na spana i ostyluj tak jak potrzebujesz, h1 to bardziej jako title calej strony, semantycznie nie pasi tu

Reply:
done